zeebe/qa/integration-tests/src/test/java/io/camunda/zeebe/it/clustering/FailOverReplicationTest.java

```diff
@@ -282,8 +282,9 @@ private static void configureBroker(final BrokerCfg brokerCfg) {
     final var data = brokerCfg.getData();
     data.setSnapshotPeriod(SNAPSHOT_PERIOD);
 
-    data.setLogSegmentSize(DataSize.ofKilobytes(8));
+    final var maxSize = DataSize.ofKilobytes(16);
```

Comment:
Due to the changes in `io.camunda.zeebe.engine.processing.bpmn.event.EndEventProcessor.NoneEndEventBehavior` class related to Execution Listener support, an additional command (END_EVENT, COMPLETE_ELEMENT) is now being written. 
This resulted in an extra entry in `RaftLogEntry.UnserializedApplicationEntry#entries`,
causing the previous limit of 8kb to be exceeded, so I had to increase it to prevent the `Not enough space to write record` error.